docker/stacks/stacks-regtest-miner.toml

```diff
@@ -70,7 +70,7 @@ events_keys = [
 
 # Add sbtc-signer as an event observers
 [[events_observer]]
-endpoint = "host.docker.internal:8801"
+endpoint = "sbtc-signer-1:8801"
```

Comment:
With this change we cannot run devenv with local signers (running them via `./signers run 3`)